llvm/lib/Target/AArch64/AsmParser/AArch64AsmParser.cpp

```diff
@@ -768,9 +768,7 @@ class AArch64Operand : public MCParsedAsmOperand {
     if (!MCE)
       return true;
     int64_t Val = MCE->getValue();
-    if (Val & 0x3)
-      return false;
-    return (Val >= -(0x2000000 << 2) && Val <= (0x1ffffff << 2));
+	return !(Val & 0x3);
```

Comment:
this function returns bool, so could you please change this to:

    return ((Val & 0x3) == 0);

also, please use 2 spaces for indentation, so this aligns with above code.

same for the changes below, thanks.